pandas/tests/dtypes/test_inference.py

```diff
@@ -577,6 +577,21 @@ def test_integers(self):
         result = lib.infer_dtype(arr, skipna=True)
         assert result == "integer"
 
+    @pytest.mark.parametrize(
+        "arr, skipna",
+        [
+            (np.array([1, 2, np.nan, np.nan, 3], dtype="O"), False),
```

Comment:
Don't forget to drop the `expected` fixture from your parameterization setup!

Reply:
Oh, right, sorry, sure